Tests/MapboxMapsTests/Annotations/Generated/CircleAnnotationManagerTests.swift

```diff
@@ -493,7 +493,6 @@ final class CircleAnnotationManagerTests: XCTestCase, AnnotationInteractionDeleg
         expectation = nil
     }
 
-
```

Comment:
Could you update the template as well? Otherwise these line will reappear next time annotation code is regenerated.

Reply:
yep, will do